doc/adoc/user_guide.adoc

```diff
@@ -194,6 +194,14 @@ be debugged.
 [listing]
 debug: true|false
 
+Configure Reboot Method::
+By default, the migration system will invoke `kexec` to boot back into the
+host system once migration is complete.  If this is in any way problematic,
+a regular `reboot` can be requested by setting `use_reboot: true`.
+
+[listing]
+use_reboot: true|false
```

Comment:
Maybe use "use_reboot_exec" to be more explicit.

Reply:
I agree the name could be misinterpreted. We can also think of something like

```
soft_reboot: true|false
```

if true, use kexec if false hardware reboot the machine. A full explanation of the option is part of the documentation.

From a code perspective it looks all good to me, thanks :+1: